models/fetchReviews.model.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection');
+
+function fetchReviews() {
+    return db.query(`SELECT reviews.*, CAST(COUNT(comments.review_id) AS INT) AS comment_count FROM reviews LEFT JOIN comments on reviews.review_id = comments.review_id GROUP BY reviews.review_id ORDER BY reviews.created_at DESC;`).then((review) => {
```

Comment:
Small thing, but the ticket doesnt want the review_body to be returned. At the moment, you're returning it as part of the data. COuld youl refactor this to specifically take data from the correct columns

Reply:
Good shout! Not something I noticed but certainly something to be avoided here also. Have changed it now, much cleaner! Thanks